extensions/webirc.md

```diff
@@ -21,20 +21,32 @@ When a user connects through an indirect connection to the IRC server, the user'
 ## Format
 The `WEBIRC` command MUST be the first command sent from the WebIRC gateway to the IRC server and MUST be sent before capability negotiation.
 
-The `WEBIRC` command takes four parameters: `password `, `gateway`, `hostname`, and `ip`.
+The `WEBIRC` command takes four or five parameters: `password `, `gateway`, `hostname`, `ip`, and `options`.
```

Comment:
is it still a parameter if there's a `:` per the described synax:
````
WEBIRC password gateway hostname ip [:option1 option2...]
````

Reply:
yep, it's a trailing parameter